gems/pending/spec/util/mount/miq_generic_mount_session_spec.rb

```diff
@@ -2,25 +2,29 @@
 require 'util/mount/miq_generic_mount_session'
 
 describe MiqGenericMountSession do
-  context "#connect" do
-    before do
-      MiqGenericMountSession.stub(:raw_disconnect)
-      @s1 = MiqGenericMountSession.new({:uri => '/tmp/abc'})
-      @s2 = MiqGenericMountSession.new({:uri => '/tmp/abc'})
+  let(:prefix) { File.join(Dir.tmpdir, "miq_") }
```

Comment:
I believe you need `require 'tmpdir'` here too unless you want to throw it at the top of the required class.